cmd/dump.go

```diff
@@ -56,33 +57,63 @@ var dumpCmd = &cobra.Command{
 
 		fmt.Println("> Fetching policies")
 		wantedPolicies , _ := cmd.Flags().GetStringSlice("policies")
+		wantedAPIs , _ := cmd.Flags().GetStringSlice("apis")
+
 
 		policies := []objects.Policy{}
+		apis := []objects.DBApiDefinition{}
 		var errPoliciesFetch error
+		var errApisFetch error
+
+
+		if len(wantedAPIs) > 0 {
+			for _, APIID := range wantedAPIs{
+				api :=  objects.DBApiDefinition{}
+				api.APIID = APIID
+				apis = append(apis, api)
+			}
+
+		}
+		if len(wantedPolicies) > 0  {
+			for _,wantedPolicy := range wantedPolicies{
+				if !bson.IsObjectIdHex(wantedPolicy){
+					fmt.Println("Invalid selected Policiy ID:",wantedPolicy,".")
```

Comment:
Could be better to use: `fmt.Printf("Invalid selected policy ID: %s.\n", wantedPolicy)`

Reply:
solved